component/common/Sidebar.tsx

```diff
@@ -0,0 +1,95 @@
+import { useEffect, useState, useRef } from 'react';
+import styled from 'styled-components';
+import Image from 'next/image';
+
+export const Sidebar = () => {
+  const [isMenuOpen, setIsMenuOpen] = useState<boolean>(false);
+
+  const imgPath = '/images/button/sidebar-button.svg';
+
+  const side = useRef<HTMLInputElement>(null);
+  const sideBtn = useRef<HTMLInputElement>(null);
+
+  const handleClose = event => {
+    const sideChildren = side.current?.contains(event.target);
+    const sideBtnChildren = sideBtn.current?.contains(event.target);
```

Comment:
```suggestion
  const handleClose = (event: MouseEvent) => {
    const target = event.target as HTMLButtonElement;
    const sideChildren = side.current?.contains(target);
    const sideBtnChildren = sideBtn.current?.contains(target);
```
https://freshman.tech/snippets/typescript/fix-value-not-exist-eventtarget/
이거 어떤데

Reply:
오 저거 type 다음에 넣야지 했는데 굳굳